CONTRIBUTING.md

```diff
@@ -19,25 +19,43 @@ reported the issue. Please try to include as much information as you can. Detail
 * Any modifications you've made relevant to the bug
 * Anything unusual about your environment or deployment
 
+To make it easier for you, we already provide an issue template when creating an 
+issue. It will be of great use to us if you can follow the template and capture as much detail as possible.
+
 
 ## Contributing via Pull Requests
 Contributions via pull requests are much appreciated. Before sending us a pull request, please ensure that:
 
-1. You are working against the latest source on the *master* branch.
+1. You are working against the latest source on the *development* branch.
 2. You check existing open, and recently merged, pull requests to make sure someone else hasn't addressed the problem already.
 3. You open an issue to discuss any significant work - we would hate for your time to be wasted.
 
```

Comment:
Maybe add a `set up` tap and ask developer to make sure they installed NodeJs, Npm, typescript, git and VisualStudioCode.

Reply:
Added in the latest commit.